changelogs/fragments/win_firewall_action.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- win_firewall - Support inbound, outbound connection in windows firewall.
```

Comment:
I think this needs to be expanded a bit more to state this is the default action for inbound/outbound connection